### PR TITLE
Support for Calice

### PR DESCRIPTION
**Pull request by Jels02**
## Description

Add support for Calice, a 75% Alice by Qwertlekeys

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/qwertlekeys/calice/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Calice
+
+![Calice](https://i.imgur.com/plQUTqs.jpg)
```
Comment (drashna) [inline keyboards/qwertlekeys/calice/readme.md:3]: This image was removed?

File: keyboards/qwertlekeys/calice/readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Calice
+
+![Calice](https://i.imgur.com/5Xsw1oj.jpg)
```
Comment (drashna) [inline keyboards/qwertlekeys/calice/readme.md:3]:
```suggestion
![Calice](https://i.imgur.com/5Xsw1ojl.jpg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/qwertlekeys/calice/info.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
+{
+    "keyboard_name": "Calice", 
+    "url": "", 
+    "maintainer": "Jels",
```
Comment (fauxpark) [inline keyboards/qwertlekeys/calice/info.json:4]:
```suggestion
    "keyboard_name": "Calice", 
    "manufacturer": "QwertleKeys",
    "url": "", 
    "maintainer": "Jels", 
    "usb": {
        "vid": "0x716B",
        "pid": "0x0001",
        "device_version": "0.0.1"
    },
```

File: keyboards/qwertlekeys/calice/config.h [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor paramater */
+#define VENDOR_ID       0x716b     //qk (qwertlekeys)
+#define PRODUCT_ID      0x0001     //Calice
+#define DEVICE_VER      0x0001      
+#define MANUFACTURER    QwertleKeys //pcb by jels
+#define PRODUCT         Calice
+
```
Comment (fauxpark) [inline keyboards/qwertlekeys/calice/config.h:27]:
```suggestion
```

File: keyboards/qwertlekeys/calice/rules.mk [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# MCU name
+MCU = atmega32u4
+
+BOOTLOADER = atmel-dfu
```
Comment (fauxpark) [inline keyboards/qwertlekeys/calice/rules.mk:4]:
```suggestion
# Bootloader selection
BOOTLOADER = atmel-dfu
```

File: keyboards/qwertlekeys/calice/rules.mk [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no 	    # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight. - We have custom RGB underglow
+ENCODER_ENABLE = yes		# For Knob support
```
Comment (fauxpark) [inline keyboards/qwertlekeys/calice/rules.mk:17]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
ENCODER_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/qwertlekeys/calice/config.h (hunk 4 above)
Comment (Jels02) [reply keyboards/qwertlekeys/calice/config.h:27]: Is this the new way of declaring device info?

Comment (fauxpark) [reply keyboards/qwertlekeys/calice/config.h:27]: Yes.

File: keyboards/qwertlekeys/calice/rules.mk [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no 	    # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable RGB underglow
+ENCODER_ENABLE = yes		# For Knob support
```
Comment (fauxpark) [inline keyboards/qwertlekeys/calice/rules.mk:18]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**